election-api-java/build.gradle

```diff
@@ -1,12 +1,12 @@
 plugins {
-	id 'org.springframework.boot' version '2.4.6'
-	id 'io.spring.dependency-management' version '1.0.11.RELEASE'
-	id 'java'
+	id 'org.springframework.boot' version '2.6.6'
 }
+apply plugin: 'io.spring.dependency-management'
```

Comment:
Why are we using the apply statement for these 2 rather than including them in the `plugins` section?

Reply:
This way it ensures that the correct / compatible dependency management & java plugins are used for the version of spring boot being used by the project & then that is the only thing to update going forwards. This is also how its done in the spring docs for getting started:

https://docs.spring.io/spring-boot/docs/current/gradle-plugin/reference/htmlsingle/

Reply:
👍